dvc/cli.py

```diff
@@ -355,6 +355,11 @@ def parse_args(argv=None):
                         default=False,
                         help='Do not prompt when removing '
                              'working directory files.')
+    pull_parser.add_argument(
+                        '-R',
+                        '--recursive',
+                        help='Pull cache for subfolders of the specified '
```

Comment:
minor: probably better to use the same term `directories` everywhere: subfolder -> _subdirectories_ of the the specified _directory_